graphs/README.md

```diff
@@ -26,12 +25,16 @@ docker build -t weaveworks/grafanalib .
 Make sure that you run the following commands from within the `graphs/` directory:
 
 ```
+cd graphs/
```

Comment:
Hm... the build commands also depends on `cd graphs`, perhaps we could just have the `cd` at the top.

If base image is already published, maybe we should move building instructions to the bottom?